File: 241026/청소당번 정하기/cleaning-numbering.js
const fs = require("fs");
let n = Number(fs.readFileSync(0).toString().trim());

let cntA = 0, cntB = 0, cntC = 0;

for (let i = 1; i <= n; i++) {
    
    if(i % 12 === 0){
        cntC++;
    } else if (i % 3 === 0) {
        cntB++;
    } else if (i % 2 === 0) {
        cntA++;
    }
}

console.log(cntA, cntB, cntC);

File: 241026/청소당번 정하기/cleaning-numbering.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    int cntA = 0, cntB = 0, cntC = 0;

    for(int i = 1; i <= n; i++) {
        if (i % 12 == 0) {
            cntC++;
        } else if (i % 3 == 0) {
            cntB++;
        } else if (i % 2 == 0) {
            cntA++;
        }
    }

    cout << cntA << " " << cntB << " " << cntC;
    return 0;
}